dvc/scm/git/backend/pygit2.py

```diff
@@ -193,15 +193,18 @@ def set_ref(
         message: Optional[str] = None,
         symbolic: Optional[bool] = False,
     ):
+        if message:
+            # pygit2 reflog message handling is bugged in 1.4.0
+            # see: https://github.com/libgit2/pygit2/issues/1061
```

Comment:
upstream PR to fix this has been merged but unsure when it will actually be available i na release.

https://github.com/libgit2/pygit2/pull/1062

Reply:
@pmrowla Let's just ask :)